cmd/qgb/relayer/cmd.go

```diff
@@ -127,14 +133,79 @@ func Command() *cobra.Command {
 			// creating the p2p querier
 			p2pQuerier := p2p.NewQuerier(dht, logger)
 			retrier := helpers.NewRetrier(logger, 5, 15*time.Second)
+
+			// checking if the provided home is already initiated
+			isInit := store.IsInit(logger, config.Home, store.InitOptions{NeedEVMKeyStore: true})
+			if !isInit {
+				logger.Info("please initialize the EVM keystore using the `relayer keys add/import` command")
+				return store.ErrNotInited
+			}
+
+			// creating the data store
```

Comment:
[comment for a different PR]

tbh, I don't even mind having super long init functions like this, because its easy enough to see the actual structure of the program, however I do think we've probably reached the point where some purely organizational abstraction could be useful. Just something to isolate relevant pieces of code and summarize them here.

Reply:
https://github.com/celestiaorg/orchestrator-relayer/issues/277